tests/profile.test.js

```diff
@@ -385,4 +401,202 @@ describe('Profile and accounts workflow:', function () {
+      assert.equal(browser.getAttribute('#new-email', 'placeholder'), 'guildenstern@rosencrantz.com')
+      assert.equal(browser.getAttribute('.current-avatar', 'src'), 'https://google.com/images/stock.jpg')
+    })
+
+    it('should not allow the user to save profile information if no new information is entered', () => {
+      createUserAndLogin(browser)
+      browser.url('http://localhost:3000/profile')
+      browser.waitForClickable('#edit-profile')
+      browser.click('#edit-profile')
+      browser.waitForClickable('.edit-profile-info')
+      browser.click('.edit-profile-info')
+      browser.waitForVisible('.edit-profile-info-modal')
+
+      assert.equal(browser.getAttribute('#save-profile-info', 'disabled'), 'true')
+    })
+
+    it('should not allow the user to save profile information if only whitespace is entered into the name or email fields', () => {
+      createUserAndLogin(browser)
+      browser.url('http://localhost:3000/profile')
+      browser.waitForClickable('#edit-profile')
+      browser.click('#edit-profile')
+      browser.waitForClickable('.edit-profile-info')
+      browser.click('.edit-profile-info')
+      browser.waitForVisible('.edit-profile-info-modal')
+      browser.waitForClickable('#new-username')
+      browser.setValue('#new-username', '        \n ')
+
+      assert.equal(browser.getAttribute('#save-profile-info', 'disabled'), 'true')
+
+      browser.waitForVisible('.edit-profile-info-modal')
+      browser.waitForClickable('#new-email')
+      browser.setValue('#new-email', '       ')
+
+      assert.equal(browser.getAttribute('#save-profile-info', 'disabled'), 'true')
+    })
+
+    it('should allow the user to update their name', () => {
+      createUserAndLogin(browser)
+      browser.url('http://localhost:3000/profile')
+      browser.waitForClickable('#edit-profile')
+
+      assert.equal(browser.getText('.header-title'), 'foobar baz')
+
+      browser.click('#edit-profile')
+      browser.waitForClickable('.edit-profile-info')
+      browser.click('.edit-profile-info')
+      browser.waitForVisible('.edit-profile-info-modal')
+      browser.waitForClickable('#new-username')
+      browser.setValue('#new-username', 'my shiny new name')
+
+      browser.waitForClickable('#save-profile-info')
+      browser.click('#save-profile-info')
+      browser.pause(1000)
+
+      assert.equal(browser.getText('.header-title'), 'my shiny new name')
+    })
+
+    it('should allow the user to update their email', () => {
+      createUserAndLogin(browser)
+      browser.url('http://localhost:3000/profile')
+      browser.waitForClickable('#edit-profile')
+
+      assert.equal(browser.getText('.profile-info-email'), 'guildenstern@rosencrantz.com')
+
+      browser.click('#edit-profile')
+      browser.waitForClickable('.edit-profile-info')
+      browser.click('.edit-profile-info')
+      browser.waitForVisible('.edit-profile-info-modal')
```

Comment:
FYI @jellegerbrandy this is where the last build failed. Builds are failing at inconsistent points, but in general the issue seems to be clicking the '.edit-profile-info' button and the modal never appearing. 

Interestingly, [this meteor forum post](https://forums.meteor.com/t/how-to-make-acceptance-tests-more-stable-using-chimp-webdriver/32259/2) details a number of meteor/chimp issues, one of which being the rendering of bootstrap modals. However, I don't think the user proposes a solution.

Does anything jump out to you about these tests that should be fixed? I am at a loss 😅

Reply:
See my comment above (about opning `edit-profile` using bootstrap modal.show())